assets/src/edit-story/components/panels/design/animation/effectChooserDropdown.js

```diff
@@ -77,16 +89,13 @@ export default function EffectChooserDropdown({
 
   return (
     <>
-      <DropDownSelect
+      <StyledDropDownSelect
+        hasAnimation={Boolean(selectedEffectTitle)}
+        activeItemLabel={selectedEffectTitle || __('None', 'web-stories')}
         aria-label={__('Animation: Effect Chooser', 'web-stories')}
         ref={selectRef}
-        onClick={() => setIsOpen(!isOpen)}
-      >
-        <DropDownTitle>
-          {selectedEffectTitle || __('Select Animation', 'web-stories')}
-        </DropDownTitle>
-        <DropdownIcon />
-      </DropDownSelect>
+        onSelectClick={() => setIsOpen(!isOpen)}
+      />
       <Popup
         anchor={selectRef}
         isOpen={isOpen}
```

Comment:
The `EffectChooser` below can be switched out to full `DropDown` with custom nodes as options in a follow-up PR.

Reply:
Follow-up issue: #6827